tools/dotnet-fssrgen/Program.fs

```diff
@@ -0,0 +1,465 @@
+namespace FSSRGen
```

Comment:
remove that, see my https://github.com/fsharp/FSharp.Compiler.Service/pull/556#issuecomment-212895651